### ISSUE TITLE
Teams version 3.13.0 is detecting bypass

### ISSUE THREAD

**Issue by elilelonek** [haoict/microsoft-detection-bypass#31]
I downgraded to the 3.13.0 version that you wrote in your release notes that is supported, but still doesn’t work, company portal works with A-Bypass.
But it seems that Teams detect the iOS version via Safari query.

please give us the solution, as in the past it already worked and stopped.

**Comment by drewbitt** [comment]
Teams 4.11.0 is supported for me.

**Comment by elilelonek** [comment]
> Teams 4.11.0 is supported for me.

I just tried this version, and it works for 1 hour.
After that the Teams application is opened a in-app safari page asked me to authenticate and then the application detect the jailbreak and prevent me from getting any new updates.

do you have any options?

**Comment by drewbitt** [comment]
Must be something in your organization requiring reauthentication that is then causing the redetect. I have not gotten that for months.

**Comment by elilelonek** [comment]
> Must be something in your organization requiring reauthentication that is then causing the redetect. I have not gotten that for months.

Yes, but the same method is bypassed the jailbreak detection by A-Bypass for the Company Portal application ( with the same window)

And already tried to enable it for Teams but not working.

**Comment by haoict** [comment]
They changed the detection mechanism, very different.
I'll need time to do some reverse engineering. No promises though.

**Comment by elilelonek** [comment]
Thank U very much, appreciate your effort!
Please update when you have any news...
Thx!

**Comment by elilelonek** [comment]
Hi,

Many thx for your amazing tweak again!
Can you please update it to support the latest versions of Teams and Outlook?

Thx!
________________________________
מאת: Hao Nguyen ***@***.***>
‏‏נשלח: Friday, November 18, 2022 9:53:40 AM
‏‏אל: haoict/microsoft-detection-bypass ***@***.***>
עותק: Eli Lelonek ***@***.***>; Author ***@***.***>
‏‏נושא: Re: [haoict/microsoft-detection-bypass] Teams version 3.13.0 is detecting bypass (Issue #31)


They changed the detection mechanism, very different.
I'll need time to do some reverse engineering. No promises though.
